### PR TITLE
Pin tf serving version to 1.4

### PR DESCRIPTION
**Pull request by lluunn**
latest doesn't work as in #510 
related #468

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/512)

### CODE REVIEW

File: components/k8s-model-server/images/Dockerfile.cpu [hunk 1]
```diff
@@ -37,17 +37,8 @@ RUN apt-get update && apt-get install -y \
...
- | tee /etc/apt/sources.list.d/tensorflow-serving.list && \
- curl https://storage.googleapis.com/tensorflow-serving-apt/tensorflow-serving.release.pub.gpg \
- | apt-key add -
-
-RUN apt-get update && apt-get install -y \
-        tensorflow-model-server && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+RUN curl -O http://storage.googleapis.com/tensorflow-serving-apt/pool/tensorflow-model-server/t/tensorflow-model-server/tensorflow-model-server_1.4.0_all.deb
+RUN dpkg -i tensorflow-model-server_1.4.0_all.deb
```
Comment (jlewi) [inline components/k8s-model-server/images/Dockerfile.cpu]: Do we also build 1.4 for GPU?

Comment (lluunn) [reply components/k8s-model-server/images/Dockerfile.cpu]: yes https://github.com/kubeflow/kubeflow/blob/master/components/k8s-model-server/images/Dockerfile.gpu#L76

**Comment by lluunn** [comment]
The test passed with build-image = true.
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/kubeflow_kubeflow/512/kubeflow-presubmit/856/
http://testing-argo.kubeflow.org/workflows/kubeflow-test-infra/kubeflow-presubmit-tf-serving-image-512-70381a9-856-6099?tab=workflow

**Pull request merged**